### ISSUE TITLE
CRL disp calc fails with Py2 

### ISSUE THREAD

**Issue by samoylv** [samoylv/WPG#92]
calculation CRL with dispersion under Py 2.7

[this line](https://github.com/samoylv/WPG/blob/feature/srw_update/wpg/srwlib.py#L2836) brings TypeError: only length-1 arrays can be converted to Python scalars

**Comment by buzmakov** [comment]
Could you please provide simple test to reproduce this error?

**Comment by samoylv** [comment]
tutorial3_beamline_focusing_CRL_disp.ipynb, running on Py 2.7 
in cell 12

**Comment by samoylv** [comment]
BTW, numpy.exp() in srwlib.py does not help

**Comment by buzmakov** [comment]
We should pass parameters for srwl_opt_setup_CRL as **list**
```python
crl_delta = list(be_data[:,1])  # Dummy numbers)
crl_attenuation_length  = list(be_data[:,2]) #Dummy numbers)
```

May be we should check it in our wrapper create_CRL_....

**Comment by samoylv** [comment]
good point. it was also fix from @CFGrote  #80 - is that the same issue?

**Comment by samoylv** [comment]
@buzmakov fixed?
